app/code/local/Dwyera/Pinpay/etc/system.xml

```diff
@@ -24,49 +24,76 @@
                             <!-- model to take the option values from -->
                             <source_model>adminhtml/system_config_source_yesno</source_model>
                             <!-- field position -->
-                            <sort_order>1</sort_order>
+                            <sort_order>10</sort_order>
                             <!-- do not show this field in store scope -->
                             <show_in_default>1</show_in_default>
                             <show_in_website>1</show_in_website>
                             <show_in_store>0</show_in_store>
                         </active>
+                        <title translate="label">
```

Comment:
@andrew-dwyer , just wanted to make it neat. Also we won't need to change sort_order values of others for inserting a new one.